server/client_manager.py

```diff
@@ -551,9 +551,14 @@ def new_client(self, transport):
         Create a new client, add it to the list, and assign it a player ID.
         :param transport: asyncio transport
         """
+        try:
+            user_id = heappop(self.cur_id)
+        except IndexError:
+            transport.write(b'BD#This server is full.#%')
```

Comment:
Unfortunately this really is the only packet for which the client allows a message to be shown before the courtroom is fully loaded.